Settings view controller and better layout for pin view controller

NinaWie opened the pull request:
Used table views to show options and labels in pin view controller and settings
Also, after selecting a pin via the search results, the map now shows the location of the selected pin

NinaWie commented:
Can now be merged (This branch was branched from `pin_color` which is not merged itself, but I'd suggest we just merge this PR and then delete `pin_color`, then we also have all changes.

jannisborn commented on PennyMe.xcodeproj/project.pbxproj:
```diff
@@ -488,13 +492,13 @@
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = PennyMe;
+				PRODUCT_BUNDLE_IDENTIFIER = PennyMe0.1;
```
Did you intentionally change the identifier here? Not sure on the implications of this but I think this should stay as it was.

NinaWie replied:
I had to change it in order to make it run on my phone.. I can test again with the previous identifier

jannisborn commented on PennyMe/Assets.xcassets/freigestellt_FMO.imageset/Contents.json:
```diff
@@ -15,7 +16,7 @@
-    "author" : "xcode"
+    "author" : "xcode",
+    "version" : 1
   }
```
Maybe this change and the updated image below caused the starting screen to break?

NinaWie replied:
Looks like only the two lines are switched, would be weird if that's the reason. No idea why the image is changed, will check later

jannisborn reviewed (changes requested):
The new features are really great but with 2 changes I'm not sure we want to merge then

NinaWie commented:
I have cleaned up the files such that the bundle id is not changed and also info.plist is left unchanged.
Also, I fixed the launch screen issue (image was not shown), but only by renaming the image files and I wasn't able to name it to something else than "test" now. I cleaned up the other image assets (there were three different FMO image sets and none of them worked) but I still have local backups of those as well.
I'd suggest to leave it like that for now and move on, even though I'm not happy with the `test.imageset`. It seems to be a well known Xcode bug that is unsolved and for now it's working like that. When we publish the app and want to have a pretty launch screen, we can take care of it again.

jannisborn reviewed (approved):
Great work 🪂
